docs/reference/changelog-r2021.md

```diff
@@ -11,6 +11,7 @@ Released on XX, XXth, 2021.
     - Increased the mouse wheel speed when zooming the 3D window ([#3565](https://github.com/cyberbotics/webots/pull/3565)).
     - Added speed vector output to GPS ([#3742](https://github.com/cyberbotics/webots/pull/3742)).
     - Added warning when attempting to add a node to an already started simulation ([#3926](https://github.com/cyberbotics/webots/pull/3926)).
+    - Added the [ColladaShapes](../guide/object-shapes.md#colladashapes) node that allows referencing Collada shapes and textures ([#3956](https://github.com/cyberbotics/webots/pull/3956)).
```

Comment:
This branch needs to be resynched with develop and the changelog entry should be added to `changelog-r2022.md` instead.